### PR TITLE
Bump commons-compress to 1.21

### PR DESCRIPTION
**Pull request by dakrone**
This updates the dependency version for the Apache commons compress project to 1.21

### CODE REVIEW

File: plugins/ingest-attachment/build.gradle [hunk 1]
```diff
@@ -55,7 +55,7 @@ dependencies {
   // MS Office
   api "org.apache.poi:poi-scratchpad:${versions.poi}"
   // Apple iWork
-  api 'org.apache.commons:commons-compress:1.19'
+  api 'org.apache.commons:commons-compress:1.21'
```
Comment (mark-vieira) [inline plugins/ingest-attachment/build.gradle:58]: As far as I can see this is the only use of this dependency in actual production code. Everything else is either build logic or test code. Does that seem right?

**Review by mark-vieira — APPROVED** [review]

File: plugins/ingest-attachment/build.gradle (hunk 1 above)
Comment (dakrone) [reply plugins/ingest-attachment/build.gradle:58]: Yep that's correct

**Review by rjernst — APPROVED** [review]
LGTM

**Comment by dakrone** [comment]
@elasticmachine update branch

**Comment by dakrone** [comment]
@elasticmachine update branch

**Pull request merged**